scene/change-atmosphere-effect/src/main/java/com/esri/samples/change_atmosphere_effect/ChangeAtmosphereEffectSample.java

```diff
@@ -80,39 +77,66 @@ public void start(Stage stage) {
       Camera camera = new Camera(64.416919, -14.483728, 100, 318, 105, 0);
       sceneView.setViewpointCamera(camera);
 
-      // create a control panel
-      VBox controlsVBox = new VBox(6);
-      controlsVBox.setBackground(new Background(new BackgroundFill(Paint.valueOf("rgba(0, 0, 0, 0.3)"),
-          CornerRadii.EMPTY, Insets.EMPTY)));
-      controlsVBox.setPadding(new Insets(10.0));
-      controlsVBox.setMaxSize(260, 110);
-      controlsVBox.getStyleClass().add("panel-region");
-
-      // create buttons to set each atmosphere effect
-      Button noAtmosphereButton = new Button("No Atmosphere Effect");
-      Button realisticAtmosphereButton = new Button ("Realistic Atmosphere Effect");
-      Button horizonAtmosphereButton = new Button ("Horizon Only Atmosphere Effect");
-      noAtmosphereButton.setMaxWidth(Double.MAX_VALUE);
-      realisticAtmosphereButton.setMaxWidth(Double.MAX_VALUE);
-      horizonAtmosphereButton.setMaxWidth(Double.MAX_VALUE);
-
-      noAtmosphereButton.setOnAction(event -> sceneView.setAtmosphereEffect(AtmosphereEffect.NONE));
-      realisticAtmosphereButton.setOnAction(event -> sceneView.setAtmosphereEffect(AtmosphereEffect.REALISTIC));
-      horizonAtmosphereButton.setOnAction(event -> sceneView.setAtmosphereEffect(AtmosphereEffect.HORIZON_ONLY));
-
-      // add buttons to the control panel
-      controlsVBox.getChildren().addAll(noAtmosphereButton, realisticAtmosphereButton, horizonAtmosphereButton);
+      // create combo box for the atmosphere effects
+      ComboBox<AtmosphereEffect> comboBox = new ComboBox<>();
+      comboBox.getItems().addAll(AtmosphereEffect.NONE,
+        AtmosphereEffect.REALISTIC, AtmosphereEffect.HORIZON_ONLY);
+
+      // show the name of the atmosphere effects in the combo box
+      comboBox.setConverter(new ComboBoxStringConverter());
+      comboBox.setCellFactory(comboBox2 -> new AtmosphereEffectListCell());
```

Comment:
`comboBox2` doesn't feel quite right here, let's go with the shorthand like we do elsewhere
```suggestion
      comboBox.setCellFactory(c -> new AtmosphereEffectListCell());
```